hs_access_control/models/community.py

```diff
@@ -0,0 +1,117 @@
+from django.contrib.auth.models import User, Group
+from django.db import models
+from hs_core.models import BaseResource
+from django.db.models import Q
+
+###################################
+# Communities of groups
+###################################
+
+
+class Community(models.Model):
+    """ a placeholder class for a community of groups """
+    name = models.TextField(null=False, blank=False)
+    description = models.TextField(null=False, blank=False)
+    purpose = models.TextField(null=True, blank=True)
+    auto_approve = models.BooleanField(null=False, default=False, blank=False, editable=False)
+    date_created = models.DateTimeField(editable=False, auto_now_add=True)
+    picture = models.ImageField(upload_to='community', null=True, blank=True)
+
+    @property
+    def member_groups(self):
+        return Group.objects.filter(gaccess__active=True, g2gcp__community=self)
```

Comment:
Is a group still a member of a community if group community privilege is NONE?

Reply:
Nope. If you set the group community privilege to NONE (in any way), the effect is to delete the group from the community. Thus group community privilege can only be VIEW or CHANGE. FOr this reason, it is not necessary to check for NONE. This is done in the utility functions.